presto-hive-metastore/src/main/java/com/facebook/presto/hive/MetastoreClientConfig.java

```diff
@@ -268,4 +269,17 @@ public MetastoreClientConfig setPartitionCacheColumnCountLimit(int partitionCach
         this.partitionCacheColumnCountLimit = partitionCacheColumnCountLimit;
         return this;
     }
+
+    public boolean isDeleteFilesOnTableDrop()
+    {
+        return deleteFilesOnTableDrop;
+    }
+
+    @Config("hive.metastore.thrift.delete-files-on-table-drop")
+    @ConfigDescription("Delete files on drop in case the metastore doesn't do it")
```

Comment:
nit: "Delete files on dropping table in case the metastore fails to do so"

Reply:
Done